.github/workflows/build-testing.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       arch: amd64
       tag: testing-amd64
       xdr_ref: v20.0.2
-      core_ref: v20.1.0
+      core_ref: v20.2.0rc1
```

Comment:
Can you also update the testing target in the `Makefile`?

Reply:
> Can you also update the testing target in the `Makefile`?

Nice catch! Updated!